src/docker/tests/test_dockerfile.py

```diff
@@ -42,9 +43,9 @@ def setup(self, tmpdir):
 
     def test_add_instruction(self):
         cmd = "RUN echo Hello, World!"
-        cmds_pre = self.dfile._cmds[:]  # Make a copy.
+        cmds_pre = deepcopy(self.dfile._cmds)  # Make a copy.
```

Comment:
Slice notation should be fine here because the list only contains strings. Deepcopy would be good if it contained mutable objects. But I'm fine with the changes.